keycloak/ldap_group_mapper.go

```diff
@@ -69,9 +69,10 @@ func convertFromLdapGroupMapperToComponent(ldapGroupMapper *LdapGroupMapper) *co
 		"drop.non.existing.groups.during.sync": {
 			strconv.FormatBool(ldapGroupMapper.DropNonExistingGroupsDuringSync),
 		},
-		"groups.path": {
-			ldapGroupMapper.LdapGroupsPath,
-		},
+	}
+
+	if KeycloakVersionIsGreaterThanOrEqualTo(keycloakClient, "11.0.0") {
```

Comment:
I don't think we need this, but maybe it worth it anyway as it is more explicit.